daemon/protocol/python/scoot/client_lib.py

```diff
@@ -246,3 +292,17 @@ def domain(status):
                  exit_code=status.exit_code,
                  error=status.error)
   return [domain(x) for x in resp.status]
+
+
+def stop_daemon():
+  """ Request that the daemon be Stopped
+  """
+  global _client
+  if not is_started():
+    raise ScootException(Exception("Connection not started."))
+  req = daemon_pb2.EmptyStruct()
+  try:
+    _client.StopDaemon(req)
+  except Exception:
```

Comment:
I think so - we do always hit an error - we've stopped the daemon, but the client is expecting a reply.  If we make the processing error specific, we may have to spend more time maintaining this as we upgrade to new versions of grpc/protobuf.